OpenSim/Simulation/Model/BodySet.cpp

```diff
@@ -22,71 +22,4 @@
  * -------------------------------------------------------------------------- */
 
 #include "BodySet.h"
-#include <OpenSim/Common/ScaleSet.h>
 
```

Comment:
Should this file be removed?

Reply:
Yes, I think all the `<C>Set` files can safely be removed now.